### PR TITLE
Fix files required validation

### PR DESCRIPTION
**Pull request by nobodywithbody**
My PR is a:

🐛 Bug fix

Main update on the:

Plugin

This commit fix following problems:
* is impossible set field `media` as required
* if hardcode `required` on media field validation error not display on view
* if fix display error message on media type property and add file, remove it again, then validation on required is successful checked, but field with file is empty

### CODE REVIEW

**Comment by lauriejim** [comment]
Thank you for this PR, waiting for @soupette review.

File: packages/strapi-helper-plugin/lib/src/components/InputFileWithErrors/index.js [hunk 1]
```diff
@@ -14,22 +14,39 @@ import Label from 'components/Label';
...
 import InputFile from 'components/InputFile';
 import InputSpacer from 'components/InputSpacer';
+import InputErrors from 'components/InputErrors';
 
+// Styles
 import styles from './styles.scss';
 
 class InputFileWithErrors extends React.Component {
-  state = { label: false, hasValue: false };
+  state = {errors: [], label: false, hasValue: false };
```
Comment (soupette) [inline packages/strapi-helper-plugin/lib/src/components/InputFileWithErrors/index.js]: There's a space missing before `{`

File: packages/strapi-helper-plugin/lib/src/components/InputFileWithErrors/index.js [hunk 2]
```diff
@@ -104,11 +134,16 @@ InputFileWithErrors.defaultProps = {
   multiple: false,
   style: {},
   value: [],
+  noErrorsDescription: false,
```
Comment (soupette) [inline packages/strapi-helper-plugin/lib/src/components/InputFileWithErrors/index.js]: The defaultProps needs to be sorted

File: packages/strapi-helper-plugin/lib/src/components/InputFile/index.js [hunk 3]
```diff
@@ -108,39 +123,41 @@ class InputFile extends React.Component {
...
+              ref={(input) => this.inputFile = input}
+            />
+
+            <div className={styles.buttonContainer}>
+              <i className="fa fa-plus" />
+              <FormattedMessage id="app.components.InputFile.newFile" />
+            </div>
+          </label>
+        </div>
+        {this.isVisibleDetails() && <InputFileDetails
```
Comment (soupette) [inline packages/strapi-helper-plugin/lib/src/components/InputFile/index.js]:
```
{this.isVisibleDetails() && (
  <InputFileDetails
    file={value[this.state.position] || value[0] || value}
    multiple={multiple}
    number={value.length}
    onFileDelete={this.handleFileDelete}
  />
)}
 ```

**Comment by nobodywithbody** [comment]
I ran lint, but output dont have errors. All style errors fixed. Sory, linter on my machine dont work correctly.

**Comment by soupette** [comment]
Thanks @nobodywithbody

**Pull request merged**